Source/WebKit/DerivedSources.make

```diff
@@ -796,3 +796,14 @@ module.private.modulemap : $(WK_MODULEMAP_PRIVATE_FILE)
 	unifdef $(addprefix -D, $(FEATURE_AND_PLATFORM_DEFINES)) $(addprefix -U, $(FEATURE_AND_PLATFORM_UNDEFINES)) -o $@ $< || [ $$? -eq 1 ]
 
 all : module.private.modulemap
+
+ifeq ($(USE_INTERNAL_SDK),YES)
+WEBKIT_ADDITIONS_SWIFT_FILES = \
+	WKApplicationUtilities.swift \
+#
+
+$(WEBKIT_ADDITIONS_SWIFT_FILES): %.swift : %.swift.in
+	cp -f $< $@
```

Comment:
Not sure I understand why the input file is a `.in`, since we're just copying it. We use `.in` elsewhere to imply that there's a preprocessing / templating step. Can it just be `.swift`?

Reply:
I wasn't sure if a Makefile rule like `$(WEBKIT_ADDITIONS_SWIFT_FILES): %.swift : %.swift` would work, but I'll give it a shot.

Reply:
Yeah that just results in a circular dependency that gets dropped by make. I think I could probably make it work if I explicitly included the entire path of the source file, but I think I want to depend on `VPATH` search here so that `WKApplicationUtilities.swift` in built products gets prioritized over `WKApplicationUtilities.swift` in the SDK directory correctly.